kube_controller_manager/assets/dashboards/overview.json

```
@@ -1,338 +1,911 @@
+								}
+							],
+							"yaxis": {
+								"include_zero": true,
+								"scale": "linear",
+								"label": "",
+								"min": "auto",
+								"max": "auto"
+							},
+							"markers": []
+						},
+						"layout": {
+							"x": 2,
+							"y": 0,
+							"width": 6,
+							"height": 2
+						}
+					},
+					{
+						"id": 5382860624626359,
+						"definition": {
+							"title": "Leader Election",
+							"title_size": "16",
+							"title_align": "left",
+							"type": "check_status",
+							"check": "kube_controller_manager.leader_election.status",
+							"grouping": "cluster",
+							"group_by": [],
+							"tags": [
+								"$cluster",
+								"$scope"
+							]
+						},
+						"layout": {
+							"x": 0,
+							"y": 2,
+							"width": 2,
+							"height": 2
+						}
+					},
+					{
+						"id": 5608027259222025,
+						"definition": {
+							"title": "Kube Controller Manager memory usage vs requests",
+							"title_size": "16",
+							"title_align": "left",
+							"show_legend": true,
+							"legend_layout": "horizontal",
+							"legend_columns": [
+								"avg",
+								"min",
+								"max",
+								"value",
+								"sum"
+							],
+							"time": {},
+							"type": "timeseries",
+							"requests": [
+								{
+									"formulas": [
+										{
+											"alias": "requests",
+											"formula": "query5"
+										},
+										{
+											"alias": "usage",
+											"formula": "query1"
+										}
+									],
+									"response_format": "timeseries",
+									"queries": [
+										{
+											"query": "sum:kubernetes.memory.requests{$cluster,$scope,image_name:k8s.gcr.io/kube-controller-manager} by {pod_name}",
+											"data_source": "metrics",
+											"name": "query5"
+										},
+										{
+											"query": "sum:kubernetes.memory.usage{$cluster,$scope,image_name:k8s.gcr.io/kube-controller-manager} by {pod_name}",
+											"data_source": "metrics",
+											"name": "query1"
+										}
+									],
+									"style": {
+										"palette": "dog_classic",
+										"line_type": "solid",
+										"line_width": "normal"
+									},
+									"display_type": "line"
+								}
+							],
+							"yaxis": {
+								"include_zero": true,
+								"scale": "linear",
+								"label": "",
+								"min": "auto",
+								"max": "auto"
+							},
+							"markers": []
+						},
+						"layout": {
+							"x": 2,
+							"y": 2,
+							"width": 6,
+							"height": 2
+						}
+					}
+				]
+			},
+			"layout": {
+				"x": 4,
+				"y": 0,
+				"width": 8,
+				"height": 5
+			}
+		},
+		{
+			"id": 4643450728041347,
+			"definition": {
+				"title": "Queue latency",
+				"type": "group",
+				"background_color": "green",
+				"show_title": true,
+				"layout_type": "ordered",
+				"widgets": [
+					{
+						"id": 5674732146803099,
+						"definition": {
+							"title": "Avg time an item stays in a queue before being requested (by queue)",
```

maybe be more explicit by saying we have a queue by resource kind

```suggestion
							"title": "Avg time an item stays in a queue before being requested (by resource's queue)",
```

---

As we have "queue" everywhere, I think it is clear enough, and adding that will make the name not visible on low density mode